lib/screens/multi_sig/multi_sig_create_or_join_screen.dart

```diff
@@ -60,6 +60,14 @@ class MultiSigCreateOrJoinScreen extends StatelessWidget {
                         bloc.submitMultiSigCreateOrJoin(MultiSigAddKind.join);
                       },
                     ),
+                    VerticalSpacer.large(),
+                    AccountButton(
+                        name: 'Recover',
```

Comment:
You could put these in the Strings file.

Reply:
Oops, missed some. Thanks